FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -770,77 +1116,151 @@ components:
           type: string
-          description: A network location describing where the device is
+          description: A network location describing where the component is
         systemTimeout:
           type: integer
           minimum: 1
-          description: Seconds within which a system update is expected from the client. If no update is received, the system will be considered lifeless.
+          description: Seconds within which a component update is expected from the client. If no update is received, the FRAIHMWORK system will consider the component lifeless.
         timeOfRegistration:
           type: string
           format: date-time
-          description: The time that the system was first registered
+          description: The time that the component was first registered
       required:
         - uuid
         - name
+
     State:
       type: object
       properties:
         state:
           type: string
           description: "
-          This is a high level description of the system or subsystem state. 
-          UNAVAILABLE describes a system that cannot provide its normal services, but not due to a fault with the system. This could be because the service cannot access its host system, the system is being restarted, or any other similar reason
-          STARTING_UP is for services being powered on - services may not be available yet but they will be soon
-          ONLINE describes a system with services provided
-          FAULTED describes a system that has a significant fault that is degrading service quality and/or is not meeting its performance specifications. In general, if this system is reporting faults, this state should be used in conjunction with any HIGH (and, situationally, MEDIUM) level faults "
+          This is a high level description of the component or subsystem state. <br> <br>
+          UNAVAILABLE describes a component that cannot provide its normal services, but not due to a fault with the component. This could be because the service cannot access its host component, the component is being restarted, or any other similar reason  <br> <br>
+          STARTING_UP is for services being powered on - services may not be available yet but they will be soon  <br> <br>
+          ONLINE describes a component with services provided  <br> <br>
+          FAULTED describes a component that has a significant fault that is degrading service quality and/or is not meeting its performance specifications. In general, if this component is reporting faults, this state should be used in conjunction with any HIGH (and, situationally, MEDIUM) level faults "
           enum:
           - UNAVAILABLE
           - STARTING_UP
           - ONLINE
           - FAULTED
         description:
           type: string
-          description: An optional detailed description of the system state in order to provide
+          description: An optional detailed description of the component state in order to provide
```

Comment:
So, in most cases, the ActiveComponent API is maintained by the FHAIS, for display by the TMI Display. Do we use this description to go into detail internally? Or is this for use externally, should someone want to provide detailed descriptions about ActiveComponents they are POSTing for the display?

Reply:
Some details added. This is only for flavor and doesn't affect anything internally as of writing.